src/android/SimpleCameraPreview.java

```diff
@@ -183,28 +185,26 @@ private boolean capture(Boolean useFlash, CallbackContext callbackContext) {
 
 
     private boolean disable(CallbackContext callbackContext) {
-        if (webViewParent != null) {
-            cordova.getActivity().runOnUiThread(new Runnable() {
-                @Override
-                public void run() {
-                    try {
-                        webView.getView().bringToFront();
-                        webViewParent = null;
-                        FrameLayout containerView = cordova.getActivity().findViewById(containerViewId);
-                        ((ViewGroup) containerView.getParent()).removeView(containerView);
-                    } catch(Exception e) {
-                        e.printStackTrace();
-                        callbackContext.error(e.getMessage());
-                    }
+        Runnable removeViewRunnable = new Runnable() {
+            @Override
+            public void run() {
+                webView.getView().bringToFront();
+                webViewParent = null;
+                FrameLayout containerView = cordova.getActivity().findViewById(containerViewId);
+                ((ViewGroup) containerView.getParent()).removeView(containerView);
+            }
+        };
+        RunnableFuture<Void> removeViewTask = new FutureTask<>(removeViewRunnable, null);
 
-                }
-            });
-        }
         try {
+            if (webViewParent != null) {
+                cordova.getActivity().runOnUiThread(removeViewTask);
```

Comment:
Pourquoi pas directement ?
```suggestion
                RunnableFuture<Void> removeViewTask = new FutureTask<>(
                    new Runnable() {
                        ...
                    },
                    null
                );
                cordova.getActivity().runOnUiThread(removeViewTask);
```

Reply:
c'est bon j'ai appliqué cette suggestion

Reply:
et Pourquoi pas carrément:

```suggestion
                cordova.getActivity().runOnUiThread(new FutureTask<>(
                    new Runnable() {
                        @Override
                        public void run() {
                            webView.getView().bringToFront();
                            webViewParent = null;
                            FrameLayout containerView = cordova.getActivity().findViewById(containerViewId);
                            ((ViewGroup) containerView.getParent()).removeView(containerView);
                        }
                    },
                    null
                ));
```